internal/storage/s3/init.go

```diff
@@ -0,0 +1,21 @@
+package s3
+
+import (
+	"context"
+
+	"github.com/minio/minio-go/v7"
+)
+
+type FileRepo struct {
+	mio *minio.Client
+}
+
+func NewFileRepo(ctx context.Context, mio *minio.Client) (*FileRepo, error) {
+	fr := FileRepo{mio: mio}
+	// TODO add confilt param.
+	_, err := fr.mio.BucketExists(ctx, "gohpkeeper")
```

Comment:
лучше в конфиг убрать
```suggestion
	_, err := fr.mio.BucketExists(ctx, config.Bucket)
```

Reply:
Да, все настройки перенес в конфиг.